resources/configurations/examples/change_domain.yaml

```diff
@@ -21,7 +21,7 @@ SimpleParticipant_domain0:                                                  # 3
   domain: 0                                                                 # 5
 
 ##################################
-# SIMPLE PARTICIPANT DOMAIN 1
+# SIMPLE PARTICIPANT DOMAIN 0
```

Comment:
why this change?

Reply:
:+1: Fixed in 145bc84c186a1d673f7ef83ee3308e58c264d503